linter.py

```diff
@@ -11,39 +11,17 @@
 """This module exports the Standard plugin class."""
 
 from SublimeLinter.lint import NodeLinter
-import re
 
 
 class Standard(NodeLinter):
     """Provides an interface to standard."""
 
-    syntax = ('javascript', 'html', 'javascriptnext', 'javascript 6to5', 'javascript (babel)')
     cmd = 'standard --stdin --verbose'
-    version_args = '--version'
-    version_re = r'(?P<version>\d+\.\d+\.\d+)'
-    version_requirement = '>= 3.7.2'
+    name = 'Standard'
     regex = r'^\s.+:(?P<line>\d+):(?P<col>\d+):(?P<message>.+)'
-    selectors = {
-        'html': 'source.js.embedded.html'
-    }
-
-    html_pattern = re.compile(r'(^.*\n)\s+$', re.DOTALL)
 
-    npm_name = 'standard'
     defaults = {
         'enable_if_dependency': True,
-        'disable_if_not_dependency': False
+        'disable_if_not_dependency': False,
+        'selector': 'source.js, source.jsx'
     }
-
-    def run(self, cmd, code):
-        """
-        If HTML syntax and the last line is just whitespace then remove it.
-
-        Its probably just space before closing.
-        script tag
-        """
-        if code and self.syntax == 'html':
```

Comment:
I've just read this here and thought I comment even if it's old.  

The new way to do this is `if code and self.view.match_selector(0, 'text.html.basic'):`, t.i. there is basically no new way you just use the (more powerful) Sublime Text API.  The scope `text.html.basic` is debatable, I'm no experts in the scope namings thing.  But there is also `text.html.vue` iirc and you probably want to exclude that.

Besides that, there is a faster way to check if the trailing line of a string is just whitespace, no?  Probably you don't start at char 0 but come from the end 😁.